src/modules/rover_pos_control/rover_pos_control_params.c

```diff
@@ -248,17 +249,26 @@ PARAM_DEFINE_FLOAT(GND_SPEED_TRIM, 3.0f);
 PARAM_DEFINE_FLOAT(GND_SPEED_MIN, 1.0f);
 
 /**
- * Maximum ground speed
+ * Maximum output of velocity PID controller
  *
+ * Used for limiting output of the speed PID controller (which is somewhat in m/s unit)
+ *
+ * Note: We need to figure out what the unit dimension is for the output of the Velocity PID controller,
+ * since we are mapping it directly to throttle, and it will be then *sqrt of thrust (for RPM controlled ESCs),
+ * but we are calculating based off of velocity error.
+ *
+ * Note: What is the role of the I-term? E.g. in Multicopter, the I-term of the Velocity controller accounts for
+ * constant offset force (acceleration) from external source. E.g. Wind.
+ *
+ * What is it for the boat?
```

Comment:
This is still quite confusing to me. Why do we even need this velocity PID controller's output limiter?